### DISCUSSION TITLE
Best way to enable feature gates for kublet?

### DISCUSSION THREAD

**Discussion by kinluek** [kubermatic/kubeone#3088] [General]
I want to start using KubeOne, but there is a specific feature that I need to enable for pod security which is the "UserNamespacesSupport", I see that there is no native way to do it with KubeOne. Are there any good work arounds for this?

I was going to use addons with a k8s jobs that runs a script to enable it, but this seems quite hacky. Any other ways that could be better? 

Thanks

**Comment by kron4eg** [comment]
At the moment this is not possible, please file the [feature request](https://github.com/kubermatic/kubeone/issues/new?assignees=&labels=sig%2Fcluster-management%2C+kind%2Ffeature&projects=&template=feature-request.md) issue
